### PR TITLE
[CORL-1538] add documentation for integrating with mobile

### PR DESCRIPTION
**Pull request by tessalt**
## What does this PR do?

First stab at adding a generic step-by-step guide for integrating a native mobile app. I'm hesitant to add any specific native code as there are quite a few different OS/language combinations these days and things tend to change fairly quickly:

- iOS objective C
- iOS swift with storyboards
- iOS swift with swiftUI
- react native
- android java
- android kotlin
- other hybrid web frameworks

## What changes to the GraphQL/Database Schema does this PR introduce?
none

### CODE REVIEW

File: docs/source/02-11-native-mobile.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Host your embed code (find your **Embed code** under **Configure** > **Organization** > **Site Details**) in an HTML page served over HTTPS. For example, `https://yoursitename.com/coral.html`
+2. Add the domain to the list of permitted domains for your site under **Configure** > **Organization** > **Site Details**
+3. Create a web view in your native application which points to this URL
+4. Pass an SSO access token through to the embed code to log in your user. See [Single Sign On](/talk/v5/integrating/sso/) for information on how to generate an SSO token. There are multiple ways to pass data from a native application to a web view, one method is to encode the access token in a query parameter on the URL (ex. `https://yoursitename.com/coral.html?accessToken=ssoToken`) and retrieve the token from the embed code. You can then pass through the `accessToken` option passed to `createStreamEmbed`:
```
Comment (wyattjoh) [inline docs/source/02-11-native-mobile.md]: I'd suggest making them include it in the hash instead:

```suggestion
4. Pass an SSO access token through to the embed code to log in your user. See [Single Sign On](/talk/v5/integrating/sso/) for information on how to generate an SSO token. There are multiple ways to pass data from a native application to a web view, one method is to encode the access token in a query parameter on the URL (ex. `https://yoursitename.com/coral.html#accessToken=ssoToken`) and retrieve the token from the embed code. You can then pass through the `accessToken` option passed to `createStreamEmbed`:
```

This ensure that the access token isn't logged via access logs.

**Review by wyattjoh — CHANGES_REQUESTED** [review]

File: docs/source/02-11-native-mobile.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Host your embed code (find your **Embed code** under **Configure** > **Organization** > **Site Details**) in an HTML page served over HTTPS. For example, `https://yoursitename.com/coral.html`
+2. Add the domain to the list of permitted domains for your site under **Configure** > **Organization** > **Site Details**
+3. Create a web view in your native application which points to this URL
+4. Pass an SSO access token through to the embed code to log in your user. See [Single Sign On](/talk/v5/integrating/sso/) for information on how to generate an SSO token. There are multiple ways to pass data from a native application to a web view, one method is to encode the access token in a query parameter on the URL hash (ex. `https://yoursitename.com/coral.html#accessToken=ssoToken`) and retrieve the token from the embed code. You can then pass through the `accessToken` option passed to `createStreamEmbed`:
+
+```
+  const urlParams = new URLSearchParams(window.location.search.replace("#", "?"));
```
Comment (wyattjoh) [inline docs/source/02-11-native-mobile.md]: This won't work unfortunately, you need to use `window.location.hash`.

**Review by wyattjoh — COMMENTED** [review]
This should also be added to the sidebar.

File: docs/source/02-11-native-mobile.md (hunk 2 above)
Comment (tessalt) [reply docs/source/02-11-native-mobile.md]: oops yes i tried this

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**